src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
```

Comment:
Honestly it makes me a little uncomfortable when you say "we expect nearly everything to change about Backend in the next couple of months"... are you aware of some information that I'm not aware of?  My understanding is that they're adding a backend and making maybe minor modifications, I haven't seen any code or discussion yet which is all about free range to change the entire backend system... Also if they are changing "nearly everything" shouldn't the team at least be apprised or have a say in it?  How are they ensuring that they don't break the JES backend?